repository.yaml

```diff
@@ -143,12 +144,16 @@ leakDetectionExemptions:
       - '/middlewares/tracing/zipkin/zipkin.go'
       - '/provider/kubernetes/kubernetes_test.go'
       - '/docs/user-guide/kv-config.md'
-
+    text:'PRIVATE KEY'
```

Comment:
as stated in the example here - https://confluence.tools.tax.service.gov.uk/display/DTRG/Leak+Detection+Service+%28LDS%29+-+Resolution+Steps
I think the text should be
text: 
  - 'localhostKey'
  - 'keyFile'